ckanext/dcatapchharvest/profiles.py

```diff
@@ -902,6 +904,17 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
                             format_uri
                         ))
 
+            # Set Media Type value if format does not match
+            if format_uri is None and resource_dict.get('media_type'):
+                for key, value in valid_media_types.items():
+                    if resource_dict.get('media_type') == key:
+                        media_type_uri = URIRef(value)
+                        g.add((
+                            distribution,
+                            DCT['format'],
+                            media_type_uri
+                        ))
+
```

Comment:
Further question: should the URI of the IANA media type also be set for the DCT['media_type']?
And could be the case exist, that the IANA media type format is getting by resource_dict.get('format')?
In both cases the logic needs to be adjusted.

Reply:
>  should the URI of the IANA media type also be set for the DCT['media_type']?

Yes

> could be the case exist, that the IANA media type format is getting by resource_dict.get('format')?

Not sure what you mean here?

Reply:
Added uses cases and comments for them. Can you have a look?